execute/factory.go

```diff
@@ -26,6 +25,31 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
 )
 
+const (
+	// Estimated maximum number of source chains the system will support.
+	// This value should be adjusted as we approach supporting that number of chains.
+	// Its primary purpose is to assist in defining the limits below.
+	estimatedMaxNumberOfSourceChains = 900
+
+	// maxQueryLength is set to disable queries because they are not used.
+	maxQueryLength = 0
+
+	// maxObservationLength is set to the maximum size of an observation
+	// check factory_test for the calculation
+	maxObservationLength = ocr3types.MaxMaxObservationLength
+
+	// maxOutcomeLength is set to the maximum size of an outcome
+	// check factory_test for the calculation
+	maxOutcomeLength = ocr3types.MaxMaxOutcomeLength
+
+	// maxReportLength is set to an estimate of a maximum report size
+	// check factory_test for the calculation
+	maxReportLength = ocr3types.MaxMaxReportLength
+
+	// maxReportCount is set to 1 because the commit plugin only generates one report per round.
+	maxReportCount = 1
```

Comment:
But exec takes more time to finish a round so we might have multiple commit reports? the comment is a bit confusing for me.

Reply:
This is referring to OCR reports. I'll update the comment here with something along the lines of "not to be mistaken by the ExecutePluginReport which is a single OCR report which may contain several per-chain sub-reports."